### PR TITLE
Remember debuger state, so we dont have to open it every time

### PR DESCRIPTION
**Pull request by blurymind**
This PR adds remembering if the debugger was open,  so we dont have to open it over and over again

### CODE REVIEW

File: newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js [hunk 1]
```diff
@@ -63,6 +65,13 @@ export default class LocalPreviewLauncher extends React.Component<
 
     const win = new BrowserWindow(this.state.previewBrowserWindowConfig);
     win.loadURL(`file://${this.state.previewGamePath}/index.html`);
+    win.webContents.on('devtools-opened', () => {
```
Comment (4ian) [inline newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js]: Nice synchronisation between React state and the devtools 👍

File: newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js [hunk 2]
```diff
@@ -49,6 +50,7 @@ export default class LocalPreviewLauncher extends React.Component<
     networkPreviewPort: null,
     networkPreviewError: null,
     previewGamePath: null,
+    debugToolsOpen: true,
```
Comment (4ian) [inline newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js]: Let's use the same name as the webContents and call this `devToolsOpen` - unless debug mean some other tools later but I don't think that's the case.

**Comment by 4ian** [comment]
We could almost merge as it is now and add pixi/react extensions in another PR ;) Let me know how it goes

File: newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js (hunk 2 above)
Comment (blurymind) [reply newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js]: renamed now :)

File: newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js (hunk 1 above)
Comment (blurymind) [reply newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js]: thanks, this was driving me nuts when doing the extension last time :D

**Comment by blurymind** [comment]
> We could almost merge as it is now and add pixi/react extensions in another PR ;) Let me know how it goes

Feel free to merge this when you feel is ready :) I dont mind doing the other two in another pr. Actually makes more sense

**Comment by 4ian** [comment]
Merged! Thanks for making this :)

**Comment by blurymind** [comment]
Thank you 😊 I will try to do the other two soon. Had to go to Scotland
over the weekend and had no time to work on this

**Comment by 4ian** [comment]
No worries, open the PRs when you have something for the extensions :)

**Pull request merged**